concepts/05 Widgets/ColorBox/10 Handle the Value Change Event.md

```diff
@@ -55,14 +99,14 @@ To process a new **ColorBox** value, you need to handle the value change event.
 
     <!--HTML-->
     <dx-color-box ...
-        (onValueChanged)="colorBox_valueChanged($event)">
+        (onValueChanged)="handleValueChanged($event)">
```

Comment:
I believe you have a typo here, see the screenshot. Please rename here and below
![image](https://user-images.githubusercontent.com/33040934/74924774-1056e300-53e4-11ea-9dd2-be3a339fb21f.png)

Reply:
fixed